apps/omg_child_chain/test/omg_child_chain/event_fetcher_test.exs

```diff
@@ -0,0 +1,497 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+defmodule OMG.ChildChain.EventFetcherTest do
+  use ExUnit.Case, async: true
+
+  alias OMG.ChildChain.EventFetcher
+  alias OMG.Eth.RootChain.Abi
+
+  setup do
+    table = :ets.new(String.to_atom("test-#{:rand.uniform(1000)}"), [:bag, :public, :named_table])
+    event_fetcher_name = String.to_atom("test-#{:rand.uniform(1000)}")
+
+    start_supervised(
+      {EventFetcher,
+       name: event_fetcher_name,
+       contracts: %{},
+       ets_bucket: table,
+       events: [
+         [name: :deposit_created, enrich: false],
+         [name: :exit_started, enrich: true],
+         [name: :in_flight_exit_input_piggybacked, enrich: false],
+         [name: :in_flight_exit_output_piggybacked, enrich: false],
+         [name: :in_flight_exit_started, enrich: true]
+       ]}
+    )
+
+    {:ok, %{event_fetcher_name: event_fetcher_name, table: table}}
+  end
+
+  test "the performance of event retrieving", %{table: table, event_fetcher_name: event_fetcher_name, test: test_name} do
```

Comment:
this test is slow, pls trim it down or move under some label excluded for `mix test` invocation.

```
test the performance of event retrieving (4431.8ms)
```